ivy_tests/test_ivy/test_functional/test_core/test_manipulation.py

```diff
@@ -339,22 +339,18 @@ def _basic_min_x_max(draw):
             available_dtypes=helpers.get_dtypes("numeric"),
         )
     )
-    min_val,max_val = None, None
-    if generate_min:
-        min_val = draw( helpers.array_values(dtype=dtype[0], shape=()))
+    min_val = draw(
+        st.one_of(st.just(None), helpers.array_values(dtype=dtype[0], shape=()))
+    )
     max_val = draw(
```

Comment:
Can we update `min_val` and `max_val` such that these can be arrays as well? They can be of any shape that is broadcastable to the shape of the input `x`. You should use the broadcast helper functions in the Array helpers section here --> :https://unify.ai/docs/ivy/docs/helpers/ivy_tests.test_ivy.helpers.hypothesis_helpers/ivy_tests.test_ivy.helpers.hypothesis_helpers.array_helpers.html#ivy_tests.test_ivy.helpers.hypothesis_helpers.array_helpers.mutually_broadcastable_shapes